src/Nancy/GlobalizationConfigurationExtensions.cs

```diff
@@ -13,10 +13,12 @@ public static class GlobalizationConfigurationExtensions
         /// </summary>
         /// <param name="environment">An <see cref="INancyEnvironment"/> that should be configured.</param>
         /// <param name="supportedCultureNames">Cultures that the application can accept</param>
-        public static void Cultures(this INancyEnvironment environment, IEnumerable<string> supportedCultureNames)
+        /// <param name="defaultCulture">Used to set a default culture for the application <remarks>If not specified the first supported culture is used</remarks></param>
+        public static void Cultures(this INancyEnvironment environment, IEnumerable<string> supportedCultureNames, string defaultCulture = null)
         {
             environment.AddValue(new GlobalizationConfiguration(

```

Comment:
Again this can't be the case. You cannot have a supported list of cultures be `en-US` and `sv-SE` and have it fallback to `en-GB` because that happens to be the OS culture. In the context of your application, one of the configured cultures _has_ to be able to be treated as the _default_ one.

The `ThreadCulture` should be the absolute last fallback and I can only ever see that happen if you configure `env.Cultures(null)` or `env.Cultures(Enumerable.Empty<string>())`